docs/en/Getting-Started-on-Cluster.md

```diff
@@ -0,0 +1,334 @@
+---
+layout: global
+title: Getting Started on a Cluster
+group: User Guide
+priority: 2
+---
+
+* Table of Contents
+{:toc}
+
+The [Quick Start Guide](Getting-Started.html) shows how to install Alluxio locally on a single machine.
+In this advanced quick start guide, we will install Alluxio on a 3-node Linux cluster, configure a
+AWS S3 bucket as UFS (Under File System), and perform basic tasks with the data in Alluxio.
+
+During this guide, you will:
+
+* Download and configure Alluxio in a Linux cluster
+* Configuration for AWS
```

Comment:
This should be a verb, to be consistent with other bullet points.

Reply:
done